pkgs/tools/system/thermald/default.nix

```diff
@@ -1,26 +1,48 @@
-{ stdenv, fetchFromGitHub, autoconf, automake, libtool
-, pkgconfig, dbus, dbus-glib, libxml2, autoconf-archive }:
+{ autoconf
+, autoconf-archive
+, automake
+, dbus
+, dbus-glib
+, fetchFromGitHub
+, gtk-doc
+, libevdev
+, libtool
+, libxml2
+, lzma
+, pkgconfig
+, stdenv
+, upower
+}:
 
 stdenv.mkDerivation rec {
   pname = "thermald";
-  version = "2.2";
+  version = "2.3";
 
   src = fetchFromGitHub {
     owner = "intel";
     repo = "thermal_daemon";
     rev = "v${version}";
-    sha256 = "1nrhv3bypyc48h9smj5cpq63rawm6vqyg3cwkhpz69rgjnf1283m";
+    sha256 = "0cisaca2c2z1x9xvxc4lr6nl6yqx5bww6brh73m0p1n643jgq1dl";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ autoconf automake libtool dbus dbus-glib libxml2 autoconf-archive ];
-
-  patchPhase = ''sed -e 's/upstartconfdir = \/etc\/init/upstartconfdir = $(out)\/etc\/init/' -i data/Makefile.am'';
+  nativeBuildInputs = [
+    gtk-doc
```

Comment:
If you are building documentation, you need `devdoc` in `outputs`, otherwise it will be removed:

https://github.com/NixOS/nixpkgs/blob/c12cfa428c0ee2bf2d900c32d5f0096b80236ce8/pkgs/build-support/setup-hooks/multiple-outputs.sh#L41

Also I do not see how it could build docs without `docbook-xsl-nons`.

Reply:
I am at a loss here, event with `docbook-xsl-nons` the build process does not process any files in the `docs` folder:
```
Making all in docs
make[2]: Entering directory '/build/source/docs'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/build/source/docs'
```

Reply:
Maybe it needs to be explicitly enabled with `--enable-gtk-doc` or something.

Reply:
bingo, now I need to find out how to fix gtk-doc trying to download the docbook dtd.

Reply:
Add the DTD it is trying to build to `nativeBuildInputs` (e.g. `docbook_xml_dtd_45`), it will be picked up from `$XMLCATALOG` variable set by `findXMLCatalogs` setup hook.

Reply:
Done.